### PR TITLE
Adding launchfiles, sample rosbags and improving documentation

### LINKED ISSUE #9: Add launchfile with example config
This would make it easier to get going with this package.

### PR DESCRIPTION
**Pull request by leandropineda**
# Changes

- Added description on all configuration file parameters
- Added ROS2 launchfile for running the node
- Added `sample` folder with a demo rosbag and a proper configuration file to match bag data
- Added a launchfile for playing demo rosbag and node
- Fixed unstable test (see [latest pipeline output](https://build.ros2.org/job/Fdev__ros_amr_interop__ubuntu_focal_amd64/7/console#console-section-16))
- Fixed velocity reporting (replaced `angle` by `angular`)
- Fixed `timestamp` update. Now it's updated each time the message is sent.

### CODE REVIEW

**Comment by leandropineda** [comment]
@miguelgarcia ready for review!

File: massrobotics_amr_sender_py/sample/README.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
+# Sample data for massrobotics_amr_sender
+
+Scripts, launch files, recordings and other tools for demoing and testing the `massrobotics_amr_sender` node.
+
+The `rosbag` folder contains a stripped rosbag based on `turtlebot3`. I was built relying on the [Gazebo](https://emanual.robotis.com/docs/en/platform/turtlebot3/simulation/#gazebo-simulation), [SLAM](https://emanual.robotis.com/docs/en/platform/turtlebot3/slam_simulation/) and [Navigation](https://emanual.robotis.com/docs/en/platform/turtlebot3/nav_simulation/) simulations.
```
Comment (miguelgarcia) [inline massrobotics_amr_sender_py/sample/README.md]:
```suggestion
The `rosbag` folder contains a stripped rosbag based on `turtlebot3`. It was built using on the [Gazebo](https://emanual.robotis.com/docs/en/platform/turtlebot3/simulation/#gazebo-simulation), [SLAM](https://emanual.robotis.com/docs/en/platform/turtlebot3/slam_simulation/) and [Navigation](https://emanual.robotis.com/docs/en/platform/turtlebot3/nav_simulation/) simulations.
```

File: massrobotics_amr_sender_py/sample/README.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+                   Topic: /local_plan | Type: nav_msgs/msg/Path | Count: 1453 | Serialization Format: cdr
+                   Topic: /location | Type: geometry_msgs/msg/PoseStamped | Count: 5273 | Serialization Format: cdr
+                   Topic: /mode | Type: std_msgs/msg/String | Count: 5 | Serialization Format: cdr
+                   Topic: /plan | Type: nav_msgs/msg/Path | Count: 74 | Serialization Format: cdr
+                   Topic: /troubleshooting/errorcodes | Type: std_msgs/msg/String | Count: 37 | Serialization Format: cdr
+                   Topic: /velocity | Type: geometry_msgs/msg/TwistStamped | Count: 5270 | Serialization Format: cdr
+```
+
+Messages on topics such as `/plan` and `/local_plan` were kept unchanged while messages on `/location` and `/velocity` were crafted by creating `PoseStamped` and `TwistStamped` messages using data from `Odometry` messages on topic `/odom`. The messages on the remaning topics `/battery`, `/battery_runtime`, `/load_perc_available`, `/mode` and `/troubleshooting/errorcodes` as well as all the transformation described above were generated with very basic ROS2 node that is available at `synthetic/node.py`.
```
Comment (miguelgarcia) [inline massrobotics_amr_sender_py/sample/README.md]:
```suggestion
Messages on topics such as `/plan` and `/local_plan` were kept unchanged while messages on `/location` and `/velocity` were crafted by creating `PoseStamped` and `TwistStamped` messages using data from `Odometry` messages on topic `/odom`. The messages on the remaning topics `/battery`, `/battery_runtime`, `/load_perc_available`, `/mode` and `/troubleshooting/errorcodes` as well as all the transformation described above were generated with a small ROS2 node that is available at `synthetic/node.py`.
```

File: massrobotics_amr_sender_py/sample/README.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+
+## How to run
+
+The `massrobotics_amr_sender_rosbag_launch.py` launch file describes a `massrobotics_amr_sender` node that is configured to use a configuration file that is specific for the sample rosbag, and also plays the rosbag in loop mode so the different node callbacks are executed.
```
Comment (miguelgarcia) [inline massrobotics_amr_sender_py/sample/README.md]:
```suggestion
The `massrobotics_amr_sender_rosbag_launch.py` launch file describes a `massrobotics_amr_sender` node that uses a configuration file customized for the sample rosbag, and also plays the rosbag in loop mode so the different node callbacks are executed.
```

**Review by miguelgarcia — APPROVED** [review]
LGTM, just small suggestions

**Pull request merged**